src/templates/individual-component-example.html

```diff
@@ -31,19 +31,10 @@
       // Vanilla JS Components
       var component;
       
-      if(typeof fabric != "undefined") {
+      if(typeof fabric !== "undefined") {
```

Comment:
Excellent catch! Always better to avoid type coercion :) Also, tiny nit: space before (